internal/runners/projects/projects.go

```diff
@@ -34,6 +42,19 @@ func newProjectWithOrg(name, org string, checkouts []string) projectWithOrg {
 			multilog.Error("Unable to get project %s from checkout: %v", checkout, err)
 		}
 		p.Executables = append(p.Executables, execDir)
+		lastUsed := locale.Tl("projects_last_use_unknown", "unknown")
+		if v, exists := lastUsedTimes[execDir]; exists {
+			if t, err := time.Parse(time.RFC3339, v.(string)); err == nil {
+				if time.Since(t) <= runtimeInUseCutoff && os.Getenv(constants.RuntimeInUseNoCutoffTimeEnvVarName) == "" {
```

Comment:
This logic should live in the svc imo. Maybe have it respond with two fields: last used time, and whether it is currently in use.

Reply:
That's a good idea. It cleans up the graphql too.